psycopg/psycopgmodule.c

```diff
@@ -899,6 +914,7 @@ INIT_MODULE(_psycopg)(void)
     /* set some module's parameters */
     PyModule_AddStringConstant(module, "__version__", PSYCOPG_VERSION);
     PyModule_AddStringConstant(module, "__doc__", "psycopg PostgreSQL driver");
+    PyModule_AddIntConstant(module, "__libpq_version__", PG_VERSION_NUM);
```

Comment:
I'm not 100% sure we have pg_config.h available if only libpq-dev is installed. I'm fine having PG_VERSION_NUM only for now. I'll drop _HEX on top of your patches and merge it, thank you.